app/assets/javascripts/discourse/app/controllers/history.js

```diff
@@ -215,6 +227,15 @@ export default Controller.extend(ModalFunctionality, {
     return this.currentUser && this.currentUser.get("staff");
   },
 
+  @discourseComputed()
+  canPurge() {
```

Comment:
```suggestion
  showPurgeButton() {
```

In addition to the rename, I'd like to suggest a small UX change as well. We could show the "Purge Revisions" button only after the revisions have been hidden. 

This would match what we do with permanently deleting posts and topics, the wording is slightly different there, you need to first delete and then the "Permanently Delete" button is available.